File: benchmark/results/python-invocation-script/run-003/b-baseline/output/metaengine_demo/billing/value_objects/tax_line.py
"""TaxLine value object for the billing domain."""
from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class TaxLine:
    """Immutable value object representing a tax line on an invoice."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str

File: benchmark/results/java-multilang/run-001/b-baseline/output/com/metaengine/demo/billing/value_objects/TaxLine.java
package com.metaengine.demo.billing.value_objects;

import java.time.Instant;

/** TaxLine value object for the billing domain. */
public record TaxLine(
    String id,
    Instant createdAt,
    Instant updatedAt,
    String name,
    String description
) {}
